.rubocop.yml

```diff
@@ -116,6 +116,9 @@ Naming/PredicateName:
 
 # ----- RSpec ------
 
+RSpec/NestedGroups:
+  Max: 5
```

Comment:
rubocop complained about too much example nesting in the spec i added (4 levels where the default limit is 3), and there were already a couple exceptions to this rule in `.rubocop_todo.yml`, so i just bumped this to what felt like a reasonable default and re-generated the todo file.

Reply:
👍🏻